### ISSUE TITLE
The connection to the server localhost:8080 was refused - did you specify the right host or port?

### ISSUE THREAD

**Issue by puppeteer701** [k3s-io/k3s#5558]
https://rancher.com/docs/k3s/latest/en/quick-start/

Following the instructions from above return the error `The connection to the server localhost:8080 was refused - did you specify the right host or port?`. I have it setup on GCP.

It works when I copy/paste the `/etc/rancher/k3s/k3s.yaml` from master into worker `~/.kube/config` with updated ip.

**Comment by brandond** [comment]
It sounds like you have your own kubectl binary that is not looking at /etc/rancher/k3s/k3s.yaml. If you use the kubectl that comes with K3s, it has been modified to look in that location. If you are using your own kubectl then you need to set KUBECONFIG appropriately.

**Comment by brandond** [comment]
> on the worker machine there is no /etc/rancher/k3s/k3s.yaml

That is correct. Only servers have the admin kubeconfig file present. Agents are less privileged and do not have any of the server's keys or other confidential materials.

**Comment by puppeteer701** [comment]
Ok. how does it work creating clusters and namespaces on worker instead on master?
